docs/docs/en/user-guide/adapters/fastapi/index.md

```diff
@@ -144,6 +144,28 @@ The following section presents high-level architecture diagrams for the two avai
         - Server processes and sends the next workflow message
 
 
+    #### Message Types
+
+    Before diving into the implementation, we need to learn a bit about the **message types** that FastAgency FastAPI adapter provides. Understanding these will help us handle messages in our custom client and display them properly to the users.
+
+    FastAgency tags each message sent from the server to the client over WebSocket with a `type` attribute. This helps the client differentiate between different types of messages and handle them accordingly. Let’s break them down into two categories:
+
```

Comment:
How about we put those messages that need feedback below the display only ones?

Reply:
Yeah, make sense. I will update the PR with this change.